src/forms/Form/Form.stories.tsx

```diff
@@ -125,9 +126,33 @@ const onCancel = () => {
 	alert("Cancelling form, going back to previous site");
 };
 
+const onSubmit = async (dispatch) => {

```

Comment:
This onSubmit and renderButtons seem to be repeated a lot. I think it's better if we create a "storyUtils" file on the utils folder, there we could just export the renderButtons function, that way if we need to change it for all stories we only change it once.